os/arch/arm/src/imxrt/imxrt_adc.c

```diff
@@ -727,6 +730,13 @@ static int adc_ioctl(FAR struct adc_dev_s *dev, int cmd, unsigned long arg)
 		adc_startconv(priv, true);
 		break;
 
+	// TO-DO
```

Comment:
TizenRT doesn't support ANIOC_STOP, ANIOC_TRIGGER is there.
You should remove todo comment and update it with ANIOC_TRIGGER